ca_qc_kirkland/people.py

```diff
@@ -23,19 +32,24 @@ def scrape(self):
 
             name = councillor.xpath(".//strong/text()")[0]
 
+            # Using self.get_phone does not include the extension #
             phone = (
                 councillor.xpath('.//div[contains(text(), "#")]/text()')[0]
                 .replace("T ", "")
                 .replace(" ", "-")
-                .replace(".", ",")  # correcting a typo
+                .replace(".", ",")
                 .replace(",-#-", " x")
             )
-            email = self.get_email(councillor)
+            encrypted_email = councillor.xpath('.//@href[contains(., "email")]')[0].split("#")[1]
+            email = decode_email(encrypted_email)
 
+            # cloudflare encrypts the email data
+            email = councillor.xpath(".//div/*/*/@href | .//div/*/@href | .//@href")[0]
+            decoded_email = decode_email(email.split("#", 1)[1])
```

Comment:
This makes no sense. You assign email, and then assign it again... Please clean it up.

Also, call self._cloudflare_decode instead of re-implementing decode_email here.